analog_digital_out.py

```diff
@@ -4,7 +4,9 @@
 """
 Sends ramping instructions to a MCUSB daq and live plots
 """
-import math, time, sys, os
+import math, time, sys, os, h5py, collections
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
Remove matplotlib plotting, out of the scope of this PR

Reply:
Got it! Thanks!